infrastructure/main.tf

```diff
@@ -145,9 +145,3 @@ resource "azurerm_key_vault_secret" "gw_oauth2_secret" {
   value = "${data.vault_generic_secret.gateway_oauth2_client_secret.data["value"]}"
```

Comment:
This one and the one above are copied from Hashi Vault as well; should they be kept?

Reply:
👍